platform/ios/scripts/document.sh

```diff
@@ -32,6 +32,9 @@ mkdir -p ${OUTPUT}
 
 cp platform/ios/screenshot.png "${OUTPUT}"
 
+DEFAULT_THEME="platform/darwin/docs/theme"
+THEME=${JAZZY_THEME:-$DEFAULT_THEME}
```

Comment:
This already works in full integration, from the `ios-sdk` side, since the variable that's plumbed through is `JAZZY_THEME` rather than theme - I'm renaming it to `THEME` here when I prevent default, just to make it easier to debug in the future.